modules/appgw/.header.md

```diff
@@ -0,0 +1,7 @@
+# Palo Alto Networks Virtual Network Gateway Module for Azure
+
+A terraform module for deploying a Virtual Network Gateway and its components required for the VM-Series firewalls in Azure.
+
+## Usage
+
+For usage refer to variables description, which include example for complex map of objects.
```

Comment:
woah, where is the whole documentation? examples? config snippets, etc?

Reply:
this is a complex module, I wouldn't drop it

Reply:
For VNG we don't have dedicated example, that's why I was putting in in `.header.md`. 
For Application Gateway we have a lot of examples, in which it's used, but if you want, I can prepare example for that, but I don't like this idea a lot - in future every time we need to remember not only about updating each example, but also `.header.md`.